tests/Unit/Domain/FunctionsOfTime/Test_PiecewisePolynomial.cpp

```diff
@@ -218,11 +219,17 @@ SPECTRE_TEST_CASE("Unit.Domain.FunctionsOfTime.PiecewisePolynomial",
     FunctionsOfTime::PiecewisePolynomial<deriv_order> f_of_t2 =
         serialize_and_deserialize(f_of_t);
 
-    f_of_t.update(2.0, {6.0, 0.0}, 2.1);
-    f_of_t2.update(2.0, {6.0, 0.0}, 2.1);
+    f_of_t.update(2.0, {12.0, 0.0}, 2.1);
+    f_of_t2.update(2.0, {12.0, 0.0}, 2.1);
```

Comment:
Just curious, why the switch from 6 to 12?

Reply:
It meant that the highest derivative printed is '(2,0)' rather than '(1,0)' which would have been the same value as the max deriv at t=1, so I'd just wanted to make sure it was printing the right max derivative.